snmp/tests/metrics.py

```diff
@@ -0,0 +1,153 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+TCP_COUNTS = [
+    'tcpActiveOpens',
+    'tcpPassiveOpens',
+    'tcpAttemptFails',
+    'tcpEstabResets',
+    'tcpHCInSegs',
+    'tcpHCOutSegs',
+    'tcpRetransSegs',
+    'tcpInErrs',
+    'tcpOutRsts',
+]
+TCP_GAUGES = ['tcpCurrEstab']
+UDP_COUNTS = ['udpHCInDatagrams', 'udpNoPorts', 'udpInErrors', 'udpHCOutDatagrams']
+IF_COUNTS = ['ifInErrors', 'ifInDiscards', 'ifOutErrors', 'ifOutDiscards']
+IFX_COUNTS = [
+    'ifHCInOctets',
+    'ifHCInUcastPkts',
+    'ifHCInMulticastPkts',
+    'ifHCInBroadcastPkts',
+    'ifHCOutOctets',
+    'ifHCOutUcastPkts',
+    'ifHCOutMulticastPkts',
+    'ifHCOutBroadcastPkts',
+]
+IF_RATES = [
```

Comment:
The spacing between constants is inconsistent in this module - are they grouped intentionally?  If so, adding a comment header for each group would help.

Reply:
Added!